### PR TITLE
fix(text field): add ability to style the TextField

### PR DESCRIPTION
**Pull request by JuliaHempel**
Changed the fraction wrapper to be a div so we can add styling when using the TextField.

### CODE REVIEW

File: src/components/molecules/TextField/TextField.tsx [hunk 1]
```diff
@@ -37,14 +37,14 @@ const TextField = (props: IField) => {
     />
   );
   return (
-    <>
+    <div {...rest}>
```
Comment (thegrinder) [inline src/components/molecules/TextField/TextField.tsx]: not sure if need that, you can just do that outside of the component. Also, notice that we're already spreading props over `<SizedContainer>`

Comment (lluia) [reply src/components/molecules/TextField/TextField.tsx]: @JuliaHempel I think as @thegrinder ☘️ says you can do instead:
```js
const CustomField = styled(TextField)`
  /* any CSS you see fit here */
`
```
If you need to styled the input specifically:
```js
const CustomInput = styled(InpuText)`
  /* any CSS you see fit here */
`
```

Comment (thegrinder) [reply src/components/molecules/TextField/TextField.tsx]: I was thinking more of something like that:
```
const Wrapper = styled.div`
  /* styles */
`
<Wrapper>
  <TextField />
</Wrapper>
```
if you use `styled()`, it's gonna overwrite styles of just the input wrapper, not a wrapper around the input, label, and error message. It really depends on the use case

Comment (JuliaHempel) [reply src/components/molecules/TextField/TextField.tsx]: You're right. I'll just wrap it in order to style it and close this.

**Comment by JuliaHempel** [comment]
Not needed

**Pull request closed without merging**